modules/clickhouse/src/main/java/org/testcontainers/containers/ClickHouseContainer.java

```diff
@@ -54,8 +56,8 @@ public ClickHouseContainer(final DockerImageName dockerImageName) {
     }
 
     @Override
-    protected Integer getLivenessCheckPort() {
-        return getMappedPort(HTTP_PORT);
+    public Set<Integer> getLivenessCheckPortNumbers() {
```

Comment:
Not strictly required for this PR, but during experimenting approaches I found that a handful of our modules are using the `getLivenessCheckPort` method, which has been deprecated for 4 years. I'd like to take this chance to clean up.